pkg/kubelet/config/config.go

```diff
@@ -61,7 +61,7 @@ type PodConfig struct {
 	updates chan kubetypes.PodUpdate
 
 	// contains the list of all configured sources
-	sourcesLock sync.Mutex
```

Comment:
why changing the mutex type, RWMutex has performance issues depending on the environment https://github.com/golang/go/issues/17973

Reply:
okay, I didn't know about the issue before and changed RWMutex back to mutex. Thanks